apps/home/routes.coffee

```diff
@@ -21,12 +21,16 @@ cached = require '../../lib/cached'
         #{require '../../components/block_v2/queries/block'}
       """
 
-    GQL send
+  .catch (->) # Ignore query errors
 
-  .then ({ channel: { blocks }}) ->
-    res.locals.sd.DEMO_BLOCKS = demoBlocks = shuffle blocks
+  .then (response) ->
```

Comment:
I hate that native promises don't have an `always`. So `catch` followed by another `then`